supervision/dataset/formats/pascal_voc.py

```diff
@@ -221,7 +221,7 @@ def detections_from_xml_obj(
 
         xyxy.append([x1, y1, x2, y2])
 
-        with_masks = obj.find("polygon") is not None
+        with_masks = _with_mask(obj)
```

Comment:
minor comment here for clarity `with_masks = force_masks if force_masks else _with_mask(obj)`